packages/app/src/pages/transaction/index.tsx

```diff
@@ -90,16 +91,16 @@ export const Transaction: React.FC = () => {
     },
   ];
 
-  const setupAccountInfo = async () => {
+  const setupAccountInfo = async (rpcClient: RPCClient) => {
```

Comment:
Why pass in the client as an argument, rather than use the one defined in the outer scope?

Maybe it's better to pass in the network url, then pass that to `getRPCClient(url);`, rather than passing around the entire client.